app/src/main/java/com/example/twofragmentactivity/MoviesListAdapter.kt

```diff
@@ -8,7 +8,7 @@ import androidx.recyclerview.widget.RecyclerView
 class MoviesListAdapter(private var context: Context, private val movieListOnClickInterface: MovieListOnClickInterface) :
     RecyclerView.Adapter<MoviesListViewHolder>() {
```

Comment:
now try to use ListAdapter here. 
https://developer.android.com/reference/android/support/v7/recyclerview/extensions/ListAdapter